src/IO/H5/VolumeData.cpp

```diff
@@ -31,6 +31,8 @@ VolumeData::VolumeData(const bool exists, detail::OpenGroup&& group,
                        : name + extension())
                 : name + extension()),
       version_(version),
+    // Note: volume_file_root_group_ is NOT the root group of the HDF5 file
+    // It is the group containing the volume data
```

Comment:
So why not give a more accurate variable name, perhaps `volume_data_group_`?

(Could be the current name is fine and I just lack information. But in general, a note that a variable doesn't contain the named data seems like an opportunity to improve the name...)